SpinCore_pp/ppg/generic.py

```diff
@@ -151,7 +154,8 @@ def generic(
             mytimes = np.zeros(indirect_len, dtype=times_dtype)
             time_axis = r_[0:dataPoints] / (SW_kHz * 1e3)
             ret_data = psp.ndshape(
-                [indirect_len, nScans, len(time_axis)], ["indirect", "nScans", "t"]
+                [len(time_axis),nScans,indirect_len], ["t","nScans","indirect"]
+                # note that "t" is a dimension that ends up getting split into phase cycle steps and possibly echoes as well
```

Comment:
I had changed this based on what was there previously (which was an improvement), but looking at this, the original ordering of dimensions makes more sense -- remember that the things that come *first* correspond to the *outer* loops when reading the data out of memory.

Do leave in the comment, which I find helpful.

Reply:
You need to set the order to what it was before

Reply:
sorry misunderstood your initial comment got it now